Tools/clinic/clinic.py

```diff
@@ -4378,7 +4381,7 @@ def dedent(self, line: str) -> str:
         margin = self.margin
         indent = self.indents[-1]
         if not line.startswith(margin):
-            fail('Cannot dedent, line does not start with the previous margin:')
+            fail('Cannot dedent, line does not start with the previous margin.')
```

Comment:
This would be marginally better grammar:

```suggestion
            fail('Cannot dedent; line does not start with the previous margin.')
```

Reply:
A colon would also work fine